src/views/Team/Team.test.js

```diff
@@ -0,0 +1,12 @@
+import { render } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom/cjs/react-router-dom.min';
+import Team from '../../views/Team/Team';
+
+test('renders Team details when selected', () => {
```

Comment:
This test should be `async `with an await `findyByText `to properly this the component and avoid a memory leak